devicetypes/smartthings/zigbee-white-color-temperature-bulb.src/zigbee-white-color-temperature-bulb.groovy

```diff
@@ -34,6 +34,11 @@ metadata {
 		// Generic
 		fingerprint profileId: "0104", deviceId: "010C", inClusters: "0006, 0008, 0300", deviceJoinName: "Light" //Generic Color Temperature Light
 
+		// DuraGreen
+		fingerprint profileId: "0104", deviceId: "010C", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300", outClusters: "0003, 0019", manufacturer: "DURAGREEN", model: "DG-CW-02",deviceJoinName: "DG Track Light" //DuraGreen Track Light
+		fingerprint profileId: "0104", deviceId: "010C", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300", outClusters: "0003, 0019", manufacturer: "DURAGREEN", model: "DG-CW-01",deviceJoinName: "DG CW LED Strip" //DuraGreen LED Strip
+		fingerprint profileId: "0104", deviceId: "010C", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300", outClusters: "0003, 0019", manufacturer: "DURAGREEN", model: "DG-CCT-01",deviceJoinName: "DG Down Light" //DuraGreen Down Light
```

Comment:
@LUZhanchang Please:
- change the `deviceJoinName`: to `"DG Light"` (in all 3 fingerprints)
- add space after a comma in all occurrences of: `,deviceJoinName`

Reply:
@LUZhanchang Yes, It has to be like that, because of Device Join Name policy.

Reply:
already changed, please review again. thank you.